flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -49,8 +51,6 @@ import scala.language.postfixOps
 object AkkaUtils {
```

Comment:
The parsing of Akka configs are not replaced by TimeUtils.parseDuration(..), since they are directly used for ActorSystem creation.
The only exception is "akka.ask.timeout" which is also widely used as Flink RPC timeout config. So I parsed it with TimeUtils when it is used by Flink components and keep it parsed by scala Duration when used by Akka (in AkkaUtils#getRemoteAkkaConfig).
The configs "akka.lookup.timeout" and "akka.client.timeout" are actually only used by Flink components so the parsing logics for them are replaced.

Reply:
I gave it a thought again and I think we can replace them.

The concern was that scala Duration supports infinite values like Inf, +Inf, etc. Replacing the parsing may break jobs specifying infinite values. However, since we never mentioned infinite durations in the config descriptors, I think we can treat them as finite durations and throw errors for infinite values.

Besides that, do you think we should parse the other duration configs (e.g. "akka.tcp.timeout") which were treated as string and directly set to Akka configs? Parsing them can help to do an early check, rather than expecting Akka to fail due to bad configs.

Reply:
A hotfix is added to convert akka duration configs to akka recognizable duration format.
Given that "these two mechanism don't need to stay in sync in the future" and TimeUtils currently supports non-labeled config, I think the hotfix is necessary.